### DISCUSSION TITLE
models location

### DISCUSSION THREAD

**Discussion by dallaszkorben** [openai/whisper#2345] [General]
Can you tell me where the models are downloaded when I run first time:
```
whisper ./hello.world.wav --language en --model small.en
```
I can not find it anywhere in my Ubuntu files system

And after it was downloaded, how can I update it?

**Comment by ryanheise** [comment]
It's downloaded to `$XDG_CACHE_HOME` or `~/.cache`. New versions of models get new file names which aren't already in the cache and hence get downloaded on first invocation. If you delete a model from the cache, it will get redownloaded on the next invocation of Whisper.

**Reply by dallaszkorben** [reply]
thank you very much for your answer
